dev.salaniLeo.flake.json

```
@@ -0,0 +1,32 @@
+{
+    "app-id" : "dev.salaniLeo.flake",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "flake",
+    "finish-args" : [
+        "--socket=fallback-x11",
```

```suggestion
        "--share=ipc",
        "--socket=fallback-x11",
```

this is always required for x11